ffi/newpassmanagers.cpp

```diff
@@ -53,16 +59,33 @@ LLVMPY_CreateNewModulePassManager() {
 
 API_EXPORT(void)
 LLVMPY_RunNewModulePassManager(LLVMModulePassManagerRef MPMRef,
-                               LLVMPassBuilderRef PBRef, LLVMModuleRef mod) {
+                               LLVMModuleRef mod, LLVMPassBuilderRef PBRef,
+                               LLVMPassInstrumentationCallbacksRef PICRef) {
 
     ModulePassManager *MPM = llvm::unwrap(MPMRef);
-    PassBuilder *PB = llvm::unwrap(PBRef);
     Module *M = llvm::unwrap(mod);
+    PassBuilder *PB = llvm::unwrap(PBRef);
+    PassInstrumentationCallbacks *PIC = llvm::unwrap(PICRef);
+
+    // TODO: Make these set(able) by user
+    bool DebugLogging = false;
+    bool VerifyEach = false;
 
     LoopAnalysisManager LAM;
     FunctionAnalysisManager FAM;
     CGSCCAnalysisManager CGAM;
     ModuleAnalysisManager MAM;
+
+    PrintPassOptions PrintPassOpts;
+
+#if LLVM_VERSION_MAJOR < 16
+    StandardInstrumentations SI(DebugLogging, VerifyEach, PrintPassOpts);
+#else
+    StandardInstrumentations SI(M->getContext(), DebugLogging, VerifyEach,
+                                PrintPassOpts);
+#endif
+    SI.registerCallbacks(*PIC, &FAM);
```

Comment:
Could this be:
```suggestion
    SI.registerCallbacks(PB.getPassInstrumentationCallbacks(), &FAM);
```

then this function not have to accept and unwrap a `PICRef`, as per comment below?